packages/preview-middleware/src/ui5/middleware.ts

```diff
@@ -20,9 +21,20 @@ async function initAdp(rootProject: ReaderCollection, config: AdpPreviewConfig,
     if (appVariant) {
         const adp = new AdpPreview(config, rootProject, logger);
         const layer = await adp.init(JSON.parse(await appVariant.getString()));
-        flp.config.rta = { layer };
+        flp.config.rta = {
+            layer,
+            pluginModule: `${adp.extensionScript.namespace.replace('.', '/')}/${adp.extensionScript.module}`
+        };
+
+        const ns = adp.extensionScript.namespace.split('.');
+        const path = flp.config.path.split('/');
+        path.pop();
```

Comment:
```suggestion
        const path = flp.config.path.split('/').pop();
```

Reply:
No :) Let's assume `flp.config.path = 'a/b/c'`
```ts
const path = flp.config.path.split('/');
path.pop();
```
results in `path = 'a/b'`
while
```ts
const path = flp.config.path.split('/').pop();
```
results in `path = 'c'`

Reply:
Got it 😅